Objects/weakrefobject.c

```diff
@@ -159,28 +162,43 @@ weakref_repr(PyWeakReference *self)
 {
     PyObject *name, *repr;
     _Py_IDENTIFIER(__name__);
+    PyObject* obj = PyWeakref_GET_OBJECT(self);
 
-    if (PyWeakref_GET_OBJECT(self) == Py_None)
+    if (obj == Py_None) {
         return PyUnicode_FromFormat("<weakref at %p; dead>", self);
+    }
 
+<<<<<<< HEAD
```

Comment:
This appears to be an unresolved merge conflict.

Reply:
Ooops... Sorry, I missed this one while manually backporting  :( Fixed in b68f6d6

Thanks, Ned!